pkg/controller/database/backup/cronjob.go

```diff
@@ -203,7 +202,14 @@ func postgresEnvVars(dbcr *kciv1alpha1.Database) ([]v1.EnvVar, error) {
 		v1.EnvVar{
 			Name: "GCS_BUCKET", Value: instance.Spec.Backup.Bucket,
 		},
-	}, nil
+	}
+
+	if ok, _ := dbcr.IsMonitoringEnabled(); ok {
```

Comment:
I think logging error can be helpful :)

Reply:
changed function to use already existing instance pointer.